pacemaker.spec.in

```diff
@@ -24,6 +24,7 @@
 %global cs_version %(pkg-config corosync --modversion  | awk -F . '{print $1}')
 # It has to be eventually decided whether to use Python2 or Python3
 %global py_site    %{?python_sitearch}%{!?python_sitearch:%(python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
+%global cman_allow (0%{?fedora} > 0 && 0%{?fedora} < 17) || (0%{?rhel} > 0 && 0%{?rhel} < 7)
```

Comment:
At least RHEL and CentOS should match on `%if 0%{?el6}`, will adjust.
Many other distros are not involved in RPM ecosystem -- to start with -- so this seems moot to me.